SparseGrids/tsgCoreOneDimensional.cpp

```diff
@@ -33,6 +33,7 @@
 
 #include "tsgCoreOneDimensional.hpp"
 #include "tsgIOHelpers.hpp"
+#include "tsgTPLWrappers.hpp"
```

Comment:
Do not use TPL Wrapper directly, add an eigenvalue method to the tsgLinearSolvers.hpp and tsgLinearSolvers.cpp files.

Reply:
Done.